mirgecom/boundary.py

```diff
@@ -16,14 +16,15 @@
 
 .. autoclass:: DummyBoundary
 .. autoclass:: FarfieldBoundary
+.. autoclass:: PressureOutflowBoundary
 .. autoclass:: RiemannInflowBoundary
 .. autoclass:: RiemannOutflowBoundary
-.. autoclass:: PressureOutflowBoundary
 .. autoclass:: IsothermalSlipWallBoundary
 .. autoclass:: IsothermalWallBoundary
 .. autoclass:: AdiabaticSlipBoundary
 .. autoclass:: AdiabaticNoslipWallBoundary
-.. autoclass:: LinearizedOutflowBoundary
+.. autoclass:: LinearizedOutflow2DBoundary
+.. autoclass:: LinearizedInflow2DBoundary
```

Comment:
What are the chances I could get you to name this *2D, instead of embedding the 2D down inside the name?

Reply:
LinearizedInflow2DBoundary -> LinearizedInflowBoundary2D ? If yes, done.